src/BSON/Timestamp.c

```diff
@@ -285,10 +337,12 @@ HashTable *php_phongo_timestamp_get_properties(zval *object TSRMLS_DC) /* {{{ */
 
 		MAKE_STD_ZVAL(increment);
 		ZVAL_LONG(increment, intern->increment);
+		convert_to_string(increment);
```

Comment:
Since the values are unsigned 32-bit integers, don't we still have an issue if `SIZEOF_ZEND_LONG` is 4?